app/controllers/discourse_assign/assign_controller.rb

```diff
@@ -126,17 +126,27 @@ def group_members
         SQL
       end
 
+      group_assignment_count = Topic
+        .joins("JOIN assignments a ON a.topic_id = topics.id")
+        .where(<<~SQL, group_id: group.id)
+          a.assigned_to_id = :group_id AND a.assigned_to_type = 'Group'
```

Comment:
Do we have an index on the columns here? Otherwise, filtering through all the topics is going to be expensive on a large forum.